lib/oidcstrategy.js

```diff
@@ -144,20 +146,26 @@ function onProfileLoaded(strategy, args) {
  *   - `responseMode`       For login only flows we should have token passed back to us in a POST
  *   - `validateIssuer`     if you have validation on, you cannot have users from multiple tenants sign in
  *   - `passReqToCallback`  if you want the Req to go back to the calling function for other processing use this.
- *
+ *   - `stateMaxAmount`     the max amount of state you want to save in session, default is 200
+ *   - `stateMaxAge`        the lifetime of state in session, default is 1800s = 30 minutes
+ *   - `nonceMaxAmount`     the max amount of nonce you want to save in session, default is 200
+ *   - `nonce`              the lifetime of nonce in session, default is 1800s = 30 minutes
  * Examples:
  *
  * passport.use(new OIDCStrategy({
  *   callbackURL: config.creds.returnURL,
- *   realm: config.creds.realm,
  *   scope: config.creds.scopes,
  *   clientID: config.creds.clientID,
  *   identityMetadata: config.creds.identityMetadata,
  *   responseType: config.creds.responseType,
  *   responseMode: config.creds.responseMode,
  *   validateIssuer: config.creds.validateIssuer,
  *   passReqToCallback: config.creds.passReqToCallback,
- *   loggingLevel: config.creds.loggingLevel
+ *   loggingLevel: config.creds.loggingLevel,
+ *   stateMaxAmount: config.creds.stateMaxAmount,
+ *   stateMaxAge: config.creds.stateMaxAge,
+ *   nonceMaxAmount: config.creds.nonceMaxAmount,
```

Comment:
I can understand why someone would want to configure the state and nonce max age, but they would do so as one setting, right? Why would they, for instance, set state to 1 hour but nonce to 5 hours? How do we let you configure this in Wilson?